### DISCUSSION TITLE
V2 documentation

### DISCUSSION THREAD

**Discussion by pdramos1** [vazco/uniforms#929] [Ideas]
Quick idea/question
Is it possible to get the v2 documentation available somewhere. 
We haven't migrated yet to v3 and it's useful for us to read through the v2 documentation.

**Comment by radekmie** [comment]
Hi @pdramos1! Unfortunately no, it's not hosted anywhere. If you need, you can read it in its unrendered form directly from the `docs` directory - [here's v2.6.9](https://github.com/vazco/uniforms/tree/v2.6.9/docs). And if you want, clone the repo and run it locally.
